client/components/field.js

```diff
@@ -211,6 +212,20 @@ class Field extends Component {
         }}
       />;
     }
+    if (this.props.type === 'checkbox' && this.props.name === 'fate-of-animals') {
+      return <NtsCheckBoxWithModal
```

Comment:
this coming from pages/project-version/nts/fateOfAnimal where version is set to fate-of-animal.